react-function-components/src/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+function customButton() {
```

Comment:
Components always start with a capital letter. Your function name should be: `CustomButton`.